python/torch_mlir_e2e_test/test_suite/elementwise.py

```diff
@@ -1233,6 +1233,27 @@ def forward(self, x):
 def ElementwiseDivScalarModule_basic(module, tu: TestUtils):
     module.forward(tu.rand(3, 4))
 
+# ==============================================================================
+
+
+class ElementwiseRemainderScalarModule(torch.nn.Module):
+
+    def __init__(self):
+        super().__init__()
+
+    @export
+    @annotate_args([
+        None,
+        ([-1], torch.int64, True),
```

Comment:
Once you add the logic for making the calculation with ints, also add an e2e test that checks that path